modules/mobile/app/controllers/mobile/v0/messages_controller.rb

```diff
@@ -103,6 +104,7 @@ def move
 
       def signature
         result = client.get_signature[:data]
+        # this generally logs the users name or the default empty result defined below. not sure why we care.
         Rails.logger.info('Mobile Get Message Signature Result', result:)
```

Comment:
Category: I here/data discovery
Action: remove. This is not helpful information. I'd speculate we added it because we weren't confident about how we were handling it, but everything seems fine and all this is doing is exposing user names.